src/main/kotlin/io/github/mojira/arisa/modules/CrashModule.kt

```diff
@@ -47,6 +49,20 @@ class CrashModule(
         }
     }
 
+    private fun uploadDeobfuscatedCrashes(issue: Issue, crashes: List<Pair<AttachmentUtils.TextDocument, Crash>>) {
+        val minecraftCrashesWithDeobf = crashes
+            .map { it.first.name to it.second }
+            .filter { it.second is Crash.Minecraft }
+            .map { it.first to (it.second as Crash.Minecraft).deobf }
+            .filter { it.second != null }
+            .filterNot { issue.attachments.any { attachment -> attachment.name == getDeobfName(it.first) } }
+        minecraftCrashesWithDeobf.forEach {
+            issue.addAttachment(File(getDeobfName(it.first)))
```

Comment:
Where is the crash report file? Just in the root directory of Arisa? Or is it in some subfolder? Does it get deleted after it's been uploaded?

Reply:
Yep, gets deleted in the context. We cant use Temp files from java as they are only deleted on shutdown

Reply:
Where is the `File` written though? Maybe I am overlooking it, but I did not find where this is happening.

Reply:
next PR, it was missing 👁️